### DISCUSSION TITLE
Read/write topics from your computer

### DISCUSSION THREAD

**Discussion by jmguerreroh** [Tiryoh/docker-ros2-desktop-vnc#91] [Ideas]
Is there any option to run the container, and some packages inside it and read/write topics from the host?

Thank you!

**Comment by Tiryoh** [comment]
Maybe `--network=host` option might help you if you are running the docker container on Ubuntu

```
docker run -p 6080:80 --shm-size=512m --network=host --security-opt seccomp=unconfined tiryoh/ros2-desktop-vnc:humble
```

**Reply by jmguerreroh** [reply]
It doesn't work:
```
WARNING: Published ports are discarded when using host network mode
```
Then I cannot connect through localhost:6080 for example, the Desktop is shared between my computer and the docker :(
